.travis.yml

```diff
@@ -1,19 +1,39 @@
 sudo: false
 language: python
 dist: xenial
-python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
 
-install:
-  - python setup.py develop
-  - if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then pip install -e ."[test_marshmallow]"; fi
-  - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then pip install -e ."[test]"; fi
-  - if [[ $TRAVIS_PYTHON_VERSION == 3.7 ]]; then pip install -e ."[test]"; fi
-
-script: 
-  - pytest --cov=hapic tests -vvvvv
-
-after_success:
-  - coveralls
+jobs:
+  include:
+    - stage: static-tests
+      name: fmt
+      python: "3.7"
+      before_script:
+        - pip install black isort
+      script:
+        - black --version
+        - echo -n "isort " && isort --version-number
+        - black -l 100 --exclude '/(\..*)/' --diff --check hapic tests example
+        - isort -df -c **/*.py
+    - stage: static-tests
+      name: flake8
+      python: "3.7"
+      before_script:
+        - pip install flake8
+      script:
+        - flake8 --version
+        - flake8
+    - stage: tests
+      name: tests
+      python:
+        - "3.5"
```

Comment:
multiple python version in jobs doesn't seems to work correctly, you should probably add multiple entry.

Reply:
fixed